server/index.js

```diff
@@ -15,16 +15,53 @@ app.get('/', (req, res) => {
   res.send('Hello World');
 });
 
-app.get('/api/images/:id', (req, res) => {
-  const id = req.url.slice(12);
-  db.query('SELECT * FROM images WHERE location_id = ? ORDER BY img_order ASC', [id], (err, data) => {
+// EXISTING IMPLEMENTATION
+app.get(`/api/images/:location`, (req, res) => {
+  const location = req.params.location;
+  db.query(`SELECT * FROM images WHERE location_id = ${location} ORDER BY img_order ASC`, (err, data) => {
     if (err) {
-      res.send('An error occurred');
+      res.send('An error occurred with GET request');
```

Comment:
AFAIK, it's best practice to send a status code in case of success (200) or error (500)

Reply:
@cinziaborello noted and updated!